packages/@azure/servicebus/data-plane/examples/README.md

```diff
@@ -1,10 +1,14 @@
 ## Getting started with samples ##
 
 ## Installing the library
-- Run the below to install the npm package for Azure Service Bus library
+- Run the below in your sample folder to install the npm package for Azure Service Bus library.
 ```bash
-npm install @azure/service-bus@1.0.0-preview.1
+npm install @azure/service-bus
 ```
+ If you plan to clone this repo and run the samples directly rather than copying them to your project,
```

Comment:
Would add a note that there are two ways of running the samples, one if environment is created via npm install other via git clone (if that was the intent) And then add the respective instructions?

Reply:
I am planning to remove the whole `git clone` option altogether and keep it simple.

Now that we have the Javascript samples (see https://github.com/Azure/azure-sdk-for-js/pull/1139), we will add instructions for both